### ISSUE TITLE
Add HTTP endpoint with informations required for the top up scripts into the research bots

### ISSUE THREAD

**Issue by daniel1302** [vegaprotocol/vega-market-sim#477]
We (ops) have tools to manage our internal bots, but We want to integrate the research bots and make them work on our infrastructure. To avoid rewriting scripts We would like to add one more endpoint for the research bot available on the HTTP protocol. The required endpoint is the one that returns some information about the market/assets the trader with a given `party ID` trades.

# Acceptance criteria

- A list of traders is available on the HTTP endpoint when bots are running
- Response for traders is a valid JSON output(may be other easy to parse format)
- Response provides the following data for each trader
    - Public Key
    - Asset to top-up
    - Maybe an amount of assets to the top would be nice

- It would be nice to provide the same format as below but endpoints but we can adjust our scripts for a better response structure

Example output for currently working endpoint you can find on the following URLs:

- https://traderbot-stagnet1-k8s.ops.vega.xyz/traders
- https://traderbot-fairground-k8s.ops.vega.xyz/traders
- https://traderbot-devnet1-k8s.ops.vega.xyz/traders

At the moment our script is using only the following information:
- `trader.Parameters.SettlementERC20TokenAddress`
- `trader.PubKey`

We ignore rest params during the topup procedure

For info We have topup implemented here: https://github.com/vegaprotocol/devopstools/blob/main/networktools/bots.go#L48

**Comment by cdummett** [comment]
@daniel1302

As it stands, to run market-sim bots you run a market-sim script (`vega_sim/devops/run_scenario.py`) with pre-configured set of keys and pointed at a specified network + market name, e.g.
```
python -m vega_sim.devops.run_scenario --scenario ETHUSD --network STAGNET1 --market "My Market Name 1"
```
If you want to run a second scenario on a different market, you call the script again.
```
python -m vega_sim.devops.run_scenario --scenario BTCUSD --network STAGNET1 --market "My Market Name 2"
```
We could get the scripts to spin up an HTTP endpoint but the endpoint would be per market so you would have to know the market id or market name in the first place, e.g.
```
https://researchbots-stagnet1-k8s.ops.vega.xyz/mYma4k3t1d/traders
```

The TLDR of what I am saying is if the public keys are fixed and are used for all markets, and the asset contract addresses are fixed. Could this exist in a config file rather than an HTTP endpoint..? Thoughts?
